pyproject.toml

```diff
@@ -29,9 +29,9 @@ classifiers = [
 ]
 dependencies = [
     "beautifulsoup4~=4.12",
```

Comment:
Make tags so we don't need to reference commits?

Reply:
I have no preference.